llpc/test/shaderdb/OpFunction_TestDontInline.spvasm

```diff
@@ -0,0 +1,38 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: AMDLLPC SUCCESS
```

Comment:
Would it make sense to check against the llvm IR after translation and assert the function has the `inline` attribute?

Reply:
I don't think it is necessary, whether we add inline attribute or not is not a real problem, the middleend will decide whether to inline or not. so I don't think we need to assert we should add inline attribute after translation or not.